### PR TITLE
docs: reach MVP milestone

### PR DESCRIPTION
**Pull request by maiqueb**
**What this PR does / why we need it**:
This PR updates the documentation indicating the project has reached what I consider to be the MVP.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer** *(optional)*:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,3 +151,7 @@ Below you can find information on how to push local code changes to a kind clust
 - push image to local registry: `podman push localhost:5000/maiqueb/multus-dynamic-networks-controller`
 - update manifests to use the generated image: `IMAGE_REGISTRY=localhost:5000/maiqueb make manifests`
 - deploy the controller: `kubectl apply -f manifests/dynamic-networks-controller.yaml`
+
+## Known limitations
+- the networks being hot-plugged **must** feature the `name` attribute in the `spec.config`. This is tracked in this [issue](https://github.com/maiqueb/multus-dynamic-networks-controller/issues/45).
+- the pod controller is not level driven. This is tracked in this [RFE](https://github.com/maiqueb/multus-dynamic-networks-controller/issues/48).
```
Comment (phoracek) [inline README.md:157]: Is "level driven" a well known (kubernetes) lingo?

File: README.md [hunk 2]
```diff
@@ -151,3 +151,7 @@ Below you can find information on how to push local code changes to a kind clust
 - push image to local registry: `podman push localhost:5000/maiqueb/multus-dynamic-networks-controller`
 - update manifests to use the generated image: `IMAGE_REGISTRY=localhost:5000/maiqueb make manifests`
 - deploy the controller: `kubectl apply -f manifests/dynamic-networks-controller.yaml`
+
+## Known limitations
```
Comment (phoracek) [inline README.md:155]: So we allow hot-unplug of cold-plugged (?) interfaces?

Comment (phoracek) [inline README.md:155]: Could we mention device plugin based CNIs? And that requested network resources must be available on the node, we won't reschedule if they are not

**Review by phoracek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (maiqueb) [reply README.md:157]: hmm. No. This predates all that; its a term from circuits - edge trigger vs level trigger.

File: README.md (hunk 2 above)
Comment (maiqueb) [reply README.md:155]: Currently ? Yes. 

Do we want to ?... Not sure. 

I'm inclined to say no.

Comment (maiqueb) [reply README.md:155]: Sure. 

But I do not plan on supporting that for the near future.

File: README.md (hunk 1 above)
Comment (phoracek) [reply README.md]: Oh you are such a nerd! I'm familiar with edge vs level there, but I would never make the connection in this README

File: README.md (hunk 2 above)
Comment (phoracek) [reply README.md:155]: Ack

**Review by phoracek — APPROVED** [review]
/lgtm

File: README.md (hunk 1 above)
Comment (maiqueb) [reply README.md]: Does the provided stack overflow answer help ? ... 

I am open to any other suggestions @phoracek

Comment (phoracek) [reply README.md]: perhaps "hot-plug requests done while the controller was offline won't be acted upon (or reconciled) once the controller comes back online"?

Comment (maiqueb) [reply README.md]: whoops. /me facepalms

**Pull request merged**